README.md

```diff
@@ -76,7 +76,7 @@ docker-compose exec web ./vendor/bin/grumphp run
 To run the phpunit tests:
 
 ```bash
-docker-compose exec web ./vendor/bin/phpunit
+docker-compose exec --user=$(id -u):$(id -g) web ./vendor/bin/phpunit
```

Comment:
This will not work 100% of the files.. What happens with docker is that the mapping of file ownership between the host and the container are based strictly on the User ID and the Group ID of the user that runs the command. That being said, it works in probably most of the cases as most unix systems tend to have the default user to be with GID and UID 1000. 
This is what happens in your case as well (and mine too).
If however, you were a secondary user in your machine, with GID 1001 and UID 1001 this is a missdirection as the files would either again fail to create or have the files owned by root as well.
At least i mostly think it does :D

Reply:
What do you mean with "a secondary user"? Do you mean using `su` to switch to a different user?

I just tried this by creating a second user, and switching to it, and both `id -u` and `id -g` give the expected results:

```
[pieter@thinkarch ~]$ sudo useradd testuser
[pieter@thinkarch ~]$ sudo su testuser -s /bin/bash
[testuser@thinkarch pieter]$ id -u
2002
[testuser@thinkarch pieter]$ id -g
2002
```